src/components/pages/ShoeDetailPage.vue

```diff
@@ -1,26 +1,102 @@
 <template>
-    <v-container>
-        <v-row>
-            <v-col cols="12">
-                <h1>Shoe Detail Page</h1>
-            </v-col>
-        </v-row>
-        <nav>
-            <ul>
-                <li><router-link to="/">Home</router-link></li>
-                <li><router-link to="/login">Login</router-link></li>
-                <li><router-link to="/register">Register</router-link></li>
-                <li><router-link to="/shoe/:id">Shoe Detail</router-link></li>
-                <li><router-link to="/profile">Profile</router-link></li>
-                <li><router-link to="/cart">Cart</router-link></li>
-                <li><router-link to="/checkout">Checkout</router-link></li>
-                <li><router-link to="/order/:id">Order Detail</router-link></li>
-            </ul>
-        </nav>
-    </v-container>
-    
+  <v-container class="w-75 fill-height overflow-auto align-start pa-10">
+    <div class="d-flex w-100 align-center mt-16">
+      <div class="d-flex align-center me-auto">
+        <v-icon size="x-large" icon="mdi-keyboard-backspace mr-2" />
+        <p class="text-h5 font-weight-bold">Back to Home</p>
+      </div>
+      <div class="d-flex align-center">
+        <p class="text-h6 font-weight-medium mr-2">View Cart</p>
+        <v-icon size="large" icon="mdi-cart mr-2" />
+      </div>
+    </div>
+    <v-row class="h-75">
+      <v-col cols="6" class="pa-4">
+        <div class="bg-grey-lighten-4 shoe-detail-image">
+          <v-img
+            aspect-ratio="16/9"
+            cover
+            src="/src/assets/shoe.png"
+            class="mx-auto"
+          />
+          <v-img
+            aspect-ratio="16/9"
+            cover
+            src="/src/assets/nike-logo.png"
+            class="mx-auto logo-overlay"
+            height="100"
+            width="100"
+          />
+        </div>
+      </v-col>
+      <v-col cols="6">
+        <div class="mt-12">
+          <p class="text-h4 font-weight-black">Nike Shoe Name 15</p>
+          <p class="text-h5 font-weight-medium text-grey-darken-2 mt-2">
+            ₱ 5,000
+          </p>
+        </div>
+        <div class="mt-6">
+          <p class="text-body-1 text-grey-darken-1">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+            aliquip ex ea commodo consequat.
+          </p>
+          <p class="text-body-2 font-weight-bold mt-3">
+            Men's Basketball Shoes
+          </p>
+        </div>
+        <div class="mt-8">
+          <p class="text-body-2 font-weight-medium">Color Variations :</p>
+          <div class="d-flex mt-2">
+            <ColorVariant :color-hex="'#000000'" />
```

Comment:
@balanejacob If the value is a string (instead of a variable), it can be passed to the component directly like this:
```vue
<ColorVariant color-hex="#000000" />
```